perfdash/www/index.html

```diff
@@ -11,17 +11,26 @@ <h1 class="md-display-2">Performance Dashboard</h1>
         <!-- TODO: Need a better logo here -->
         <a href="https://k8s.io"><img src="k8s_logo.png" style="width:84; height: 84" /></a>
       </md-toolbar>
-      <div id="content" class="md-whiteframe-z2" style="padding: 20px">
+      <div id="content" class="md-whiteframe-z2" style="padding: 20px;">
         <div style="width: 200px; display: inline-block;">
           <md-input-container>
-            <md-select placeholder="JobName" ng-model="controller.jobName"  ng-change="controller.jobNameChanged()">
+            <md-select ng-model="controller.jobName" ng-change="controller.jobNameChanged()">
               <md-option ng-value="jobName" ng-repeat="jobName in controller.jobNames">
                 {{jobName}}
               </md-option>
             </md-select>
           </md-input-container>
         </div>
 	<div style="width: 200px; display: inline-block;">
+          <md-input-container>
+            <md-select placeholder="ComponentName" ng-model="controller.componentName"  ng-change="controller.componentNameChanged()">
```

Comment:
Could you rename this to be `MetricCategory` instead of `ComponentName` to be more generic?

Reply:
I will do this. However, I think `TestCategoryName` will be a better name - we are having `TestName` as the lowest level selector.

Reply:
I'm not sure because this is not the category for the test, but category for the metric.

Reply:
Sure, that makes sense.